src/gui/import_export.rs

```diff
@@ -0,0 +1,111 @@
+use crate::tungsten_halogen::reference_from_filament_temp;
+use crate::{ThreadId, ThreadResult};
+use egui::{Button, Context, Slider};
+
+pub struct ImportExportWindow {
+    tungsten_filament_temp: u16,
+}
+
+impl ImportExportWindow {
+    pub fn new() -> Self {
+        Self {
+            tungsten_filament_temp: 2800,
+        }
+    }
+
+    pub fn draw(
+        &mut self,
+        ctx: &Context,
+        config: &mut crate::config::SpectrometerConfig,
+        spectrum_container: &mut crate::spectrum::SpectrumContainer,
```

Comment:
As I pointed out in #7, the architecture should probably be reworked so the UI does not own the spectrum container. Then this argument could instead be something else that allows triggering a CSV export.